package.json

```diff
@@ -48,8 +49,7 @@
     "html-loader": "^3.1.0",
     "html-webpack-plugin": "^5.5.0",
     "mkdirp": "^1.0.4",
-    "node-sass": "^7.0.0",
```

Comment:
and this package is removed? Actually I fired an [issue](https://github.com/hypertrons/hypertrons-crx/issues/326) to plan to remove node-sass, can you provide some info?

Reply:
i guess i was facing the sass error due to this reason, thats why i did yarn add sass should i remove sass and try after adding node-sass?